typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoPZETHWarpConfig.ts

```diff
@@ -31,26 +30,140 @@ const pzEthValidators = {
+    );
+  }
+
+  const tokenConfig = Object.fromEntries<HypTokenRouterConfig>(
+    await Promise.all(
+      chainsToDeploy.map(
+        async (chain): Promise<[string, HypTokenRouterConfig]> => {
+          const addresses = await registry.getChainAddresses(chain);
+          assert(addresses, 'No addresses in Registry');
+          const { mailbox } = addresses;
+
+          const mailboxContract = Mailbox__factory.connect(
+            mailbox,
+            multiProvider.getProvider(chain),
+          );
+          const defaultHook = await mailboxContract.defaultHook();
+          const ret: [string, HypTokenRouterConfig] = [
+            chain,
+            {
+              isNft: false,
+              type:
+                chain === lockboxChain
+                  ? TokenType.XERC20Lockbox
+                  : TokenType.XERC20,
+              token: chain === lockboxChain ? lockbox : xERC20,
+              owner: pzEthSafes[chain],
+              gas: warpRouteOverheadGas,
+              mailbox,
+              interchainSecurityModule: {
+                type: IsmType.AGGREGATION,
+                threshold: 2,
+                modules: [
+                  {
+                    type: IsmType.ROUTING,
+                    owner: pzEthSafes[chain],
+                    domains: buildAggregationIsmConfigs(
+                      chain,
+                      chainsToDeploy,
+                      pzEthValidators,
+                    ),
+                  },
+                  {
+                    type: IsmType.FALLBACK_ROUTING,
+                    domains: {},
+                    owner: pzEthSafes[chain],
+                  },
+                ],
+              },
+              hook: getRenzoHook(defaultHook, chain),
+              proxyAdmin: existingProxyAdmins[chain],
+            },
+          ];
+
+          return ret;
+        },
+      ),
+    ),
+  );
+
+  return tokenConfig;
 };
 
-export const getRenzoPZETHWarpConfig = async (): Promise<
+export const getRenzoPZETHWarpConfigStaging = async (): Promise<
   ChainMap<HypTokenRouterConfig>
 > => {
+  const lockbox = '0xbC5511354C4A9a50DE928F56DB01DD327c4e56d5'; // TODO
+  const xERC20 = '0x9cb41CD74D01ae4b4f640EC40f7A60cA1bCF83E7'; // TODO
+
+  // TODO DEPLOY REAL ONES
+  const stagingSafes = {
+    ethereum: '0xA9421c6F339eC414b7e77449986bE9C2Ae430C25',
+    bsc: '0xA9421c6F339eC414b7e77449986bE9C2Ae430C25',
+    arbitrum: '0xA9421c6F339eC414b7e77449986bE9C2Ae430C25',
+    optimism: '0xA9421c6F339eC414b7e77449986bE9C2Ae430C25',
+    blast: '0xA9421c6F339eC414b7e77449986bE9C2Ae430C25',
+    linea: '0xA9421c6F339eC414b7e77449986bE9C2Ae430C25',
+    base: '0xA9421c6F339eC414b7e77449986bE9C2Ae430C25',
+    mode: '0xf40b75fb85C3bEc70D75A1B45ef08FC48Db61115',
+    swell: '0xf40b75fb85C3bEc70D75A1B45ef08FC48Db61115',
+    fraxtal: '0xf40b75fb85C3bEc70D75A1B45ef08FC48Db61115',
+    zircuit: '0xf40b75fb85C3bEc70D75A1B45ef08FC48Db61115',
+  };
```

Comment:
maybe overly optimistic but I feel like we dont need to test with safes and fine to use the deployer key as owner?